@types/slick.remotemodel.d.ts

```diff
@@ -0,0 +1,5 @@
+declare namespace Slick {
```

Comment:
Same question as above, but noting that this specific file will likely not exist when the migration is complete.

Reply:
I think with this it's more dependent on the pipeline to release/publish new versions, so we should probably ask the maintainers.